global_shortcuts.go

```diff
@@ -0,0 +1,124 @@
+package astilectron
+
+import (
+	"context"
+	"sync"
+)
+
+const (
+	EventNameGlobalShortcutsCmdRegister          = "global.shortcuts.cmd.register"
+	EventNameGlobalShortcutsCmdIsRegistered      = "global.shortcuts.cmd.is.registered"
+	EventNameGlobalShortcutsCmdUnregister        = "global.shortcuts.cmd.unregister"
+	EventNameGlobalShortcutsCmdUnregisterAll     = "global.shortcuts.cmd.unregister.all"
+	EventNameGlobalShortcutsEventRegistered      = "global.shortcuts.event.registered"
+	EventNameGlobalShortcutsEventIsRegistered    = "global.shortcuts.event.is.registered"
+	EventNameGlobalShortcutsEventUnregistered    = "global.shortcuts.event.unregistered"
+	EventNameGlobalShortcutsEventUnregisteredAll = "global.shortcuts.event.unregistered.all"
+	EventNameGlobalShortcutEventTriggered        = "global.shortcuts.event.triggered"
+)
+
+type globalShortcutsCallback func()
+
+// GlobalShortcuts represents global shortcuts
+type GlobalShortcuts struct {
+	*object
+	m *sync.Mutex
+}
+
+var callbacks map[string]globalShortcutsCallback
```

Comment:
I missed it in my previous reviews, but could you move `callbacks` to the `GlobalShortcuts` struct and make sure you initialize it with `make(map[string]globalShortcutsCallback)` in `newGlobalShortcuts?`